rocketpy/environment/environment_analysis.py

```diff
@@ -2885,3 +2885,79 @@ def save(self, filename="env_analysis_dict"):
         file.write(encoded_class)
         file.close()
         print("Your Environment Analysis file was saved, check it out: " + filename)
+
+    def get_environment_object(
+        self, gravity=None, date=None, datum="SIRGAS2000", max_expected_height=80000.0
+    ):
```

Comment:
Could you change the name from `get_` to `create_`? 

`get` implies that you are just accessing an attribute or something of the sort, create would make more sense here.

After this change we can merge